script/Ciel/Day 8/08_01_CIEL08_3B.txt

```diff
@@ -828,8 +828,8 @@ After all, death permeates the entire world.
 -- Page 311, Offset 19040.
 -- 「人間が死んでも失われるものは何もありません。ミクロの視点で見れば素粒子は失われていませんし、マクロの視点で見ても、個人が死んだところで星に損失はありませんから」
 "When a human dies, nothing is actually lost.
- On a micro level there aren't any fundamental particles that disappear,
- and on a macro level the death of an individual doesn't inflict any loss on the planet itself."
+ On a micro level, there aren't any elementary particles that disappear,
```

Comment:
didn't this get fixed in Volvo's PR? I definitely left a comment about it

Reply:
I think Volvo fixed it after I made this branch, I'll peek over there and make sure I don't override any changes before I merge this later